docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
                 - INSTALL_WORKSPACE_SSH=false
                 - PUID=1000
                 - PGID=1000
-                - NODE_VERSION=stable
+                - NODE_VERSION=6
```

Comment:
Well obviously each way has it's own disadvantages.. I think we should make this clear in the docs and I agree with @philtrep let's keep this stable until it breaks!